csunplugged/config/settings/base.py

```diff
@@ -102,10 +106,43 @@
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#language-code
 LANGUAGE_CODE = "en"
 
+INCONTEXT_L10N_PSEUDOLANGUAGE = "xx-lr"
+INCONTEXT_L10N_PSEUDOLANGUAGE_BIDI = "yy-rl"
+INCONTEXT_L10N_PSEUDOLANGUAGES = (
+    INCONTEXT_L10N_PSEUDOLANGUAGE,
+    INCONTEXT_L10N_PSEUDOLANGUAGE_BIDI
+)
+
 LANGUAGES = (
     ("en", "English"),
 )
 
+if env.bool("INCLUDE_INCONTEXT_L10N", False):
+    EXTRA_LANGUAGES = [
+        (INCONTEXT_L10N_PSEUDOLANGUAGE, "In-context translations"),
+        (INCONTEXT_L10N_PSEUDOLANGUAGE_BIDI, "In-context translations (Bidi)"),
+    ]
+
+    LANGUAGES += tuple(EXTRA_LANGUAGES)
```

Comment:
This line appears unused?

Reply:
It's definitely used - the LANGUAGES variable defines the languages that are available to the end user. What were your thoughts?